### DISCUSSION TITLE
博主没有试试Commento吗？

### DISCUSSION THREAD

**Discussion by Pathsis** [YunYouJun/yunyoujun.github.io#131] [Ideas]
Disqus的开源替代品，可以在docker轻松部署。commento.io

https://app.cloud.tencent.com/detail/Commento

**Comment by YunYouJun** [comment]
没有尝试过，看起来功能还挺全。 不过可能还是部署和数据存储有点麻烦。

开源版应该还是需要自己服务器进行部署，而您所发的链接里的这个似乎还未正式开通无法使用。
![image](https://user-images.githubusercontent.com/25154432/143067446-29562d67-0ee8-4d37-a9f0-85fc18176175.png)

**Reply by Pathsis** [reply]
通过docker可以迅速部署，可以通过json备份评论，我已经建立起来了，要不您试试？😁

**Reply by YunYouJun** [reply]
docker 需要自己的服务器吧👀，有点和静态博客的初衷相违背了。还是希望可以有托管的方式。

**Reply by YunYouJun** [reply]
有点奇怪的是，仓库栏的 Issues 似乎被关闭了。
![image](https://user-images.githubusercontent.com/25154432/143069392-48e5e963-ed8e-4460-9260-8f81f89fbca4.png)

**Reply by Pathsis** [reply]
他消失好几个月了，似乎停止了对Commento的积极开发。但目前来说，它是最良心，质量最高的Disqus替代品。对隐私的保护也很强。

**Reply by Pathsis** [reply]
他主要在gitlab上工作。https://github.com/adtac/commento
